glue_astronomy/translators/regions.py

```diff
@@ -53,12 +54,24 @@ def range_to_rect(data, ori, low, high):
     return RectanglePixelRegion(PixCoord(xcen, ycen), width, height)
 
 
-def roi_subset_state_to_region(subset_state, to_sky=False):
+def roi_subset_state_to_region(subset_state, to_sky=False, override_wcs=None):
```

Comment:
This is now not used anymore.
```suggestion
def roi_subset_state_to_region(subset_state, to_sky=False):
```